### PR TITLE
Feature namespace refactor common utilities

### PR DESCRIPTION
**Pull request by DDReaper**
Overview
---
Updating Common & Utilities namespace
(code will not build until all namespaces complete!)

Changes
---
[x] Renamed base namespaces to MixedRealityToolkit.Common from Common folder
[x] Renamed base namespaces to MixedRealityToolkit.Utilities from Utilities folder
[x] Fixed copyright statement in all files
[x] Moved Attributes to Editor folder
[x] Moved Inspectors to Editor folder
[x] Removed all #if EDITOR from scripts in the Editor folders 

Outstanding
---
Build window classes left alone for now, will move when dealing with the Build namespace

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs [hunk 1]
```diff
@@ -1,15 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Reflection;
-using System.Text.RegularExpressions;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
 using UnityEngine;
-#if UNITY_EDITOR
+using MixedRealityToolkit.UtiltiesInspectors;
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs]: typo: UtiltiesInspectors -> UtilitiesInspectors

File: Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
 using UnityEditor;
-#endif
 
-namespace HoloToolkit.Unity
+namespace MixedRealityToolkit.Utilties
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs]: typo: Utilties -> Utilities (this one is in almost all of them, but I'll just put it once)

File: Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs (hunk 1 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs]: Good catch, darn Copy and Paste.
it's in a few files and darn it I missed it.  Global fix incoming

File: Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs (hunk 2 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs]: Resolved

File: Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs [hunk 3]
```diff
@@ -6,12 +6,12 @@
...
 using UnityEngine;
 
-namespace HoloToolkit.Unity
+namespace MixedRealityToolkit.Utilities.GpuTiming
 {
-	/// <summary>
-	/// Encapsulates access to GPU timing methods.
-	/// </summary>
-	public static class GpuTiming
+    /// <summary>
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs]: Looks like these tabs were turned into spaces, but not the rest of the file (still tabs). Should we go ahead and convert the rest of this file (and GpuTimingCamera.cs)?

Comment (DDReaper) [reply Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs]: I'm all in favour of not mixing tabs / spaces.  Odd it should have changed though.

File: Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs (hunk 1 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs]: Sounds good!

File: Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs (hunk 3 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs]: Ok, pushing a fix to resolve tabs for spaces across all the scripts. As that seemed to be the standard.
Might up the change file count.  But seemed to be limited to scripts that were brought in to the project externally.

**Comment by DDReaper** [comment]
Should be good now and a few more things to add to the check list for the remaining namespaces.
Will merge in the morning if there's nothing else raised.

File: Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs (hunk 3 above)
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/Utilities/Scripts/GpuTiming/GpuTiming.cs]: spaces > tabs

**Pull request merged**